arduino-ide-extension/src/browser/widgets/sketchbook/sketchbook-tree-widget.tsx

```diff
@@ -134,26 +133,37 @@ export class SketchbookTreeWidget extends FileTreeWidget {
   protected renderInlineCommands(node: TreeNode): React.ReactNode {
     if (SketchbookTree.SketchDirNode.is(node) && node.commands) {
       return Array.from(new Set(node.commands)).map((command) =>
-        this.renderInlineCommand(command.id, node)
+        this.renderInlineCommand(
+          Array.isArray(command)
+            ? command
+            : typeof command === 'string'
+            ? command
+            : command.id,
+          node
+        )
```

Comment:
I think we can handle this condition inside `renderInlineCommand` and just pass the command like this:

```suggestion
        this.renderInlineCommand(command, node)
```

Reply:
It's [called from the subclass with the command ID](https://github.com/arduino/arduino-ide/blob/e36127aed519f70bbcfaf26d8197c00559d2bf79/arduino-ide-extension/src/browser/widgets/cloud-sketchbook/cloud-sketchbook-tree-widget.tsx#L124), I did not want to change unrelated code, but I can do if you think it has value.